### PR TITLE
Adjust string translation for Chinese localization

### PR DESCRIPTION
**Pull request by tengqm**
This PR also reorders the entries in the strings table.

### CODE REVIEW

File: i18n/en.toml [hunk 1]
```diff
@@ -1,195 +1,188 @@
...
 
-[main_github_invite]
-other = "Interested in hacking on the core Kubernetes code base?"
+[layouts_docs_glossary_aka]
+other = "Also known as"
 
-[main_github_view_on]
-other = "View On GitHub"
+[layouts_docs_glossary_description]
+other = "This glossary is intended to be a comprehensive, standardized list of Kubernetes terminology. It includes technical terms that are specific to K8s, as well as more general terms that provide useful context."
```
Comment (kbhawkey) [inline i18n/en.toml]: nit: Kubernetes or K8s? Personally, I like Kubernetes.

File: i18n/en.toml [hunk 2]
```diff
@@ -1,195 +1,188 @@
...
 
-[main_page_history]
-other ="Page History"
+[layouts_docs_partials_feedback_thanks]
+other = "Thanks for the feedback. If you have a specific, answerable question about how to use Kubernetes, ask it on"
 
-[main_page_last_modified_on]
-other = "Page last modified on"
+[layouts_docs_search_fetching]
+other = "Fetching results.."
```
Comment (kbhawkey) [inline i18n/en.toml]: I understand that these text snippets are taken from a number of places on the site.
Should `layouts_docs_search_fetching` be set to `Fetching results ...`

File: i18n/en.toml (hunk 1 above)
Comment (tengqm) [reply i18n/en.toml]: Agreed. Fixed.

File: i18n/en.toml (hunk 2 above)
Comment (tengqm) [reply i18n/en.toml]: Same as above. It is a result of copy-paste. Fixed as you suggested.

**Comment by sftim** [comment]
The English-language changes look good to me.  The Chinese localization I can't comment on as I don't speak Chinese.

File: i18n/zh.toml [hunk 3]
```diff
@@ -1,59 +1,144 @@
...
 
-[main_github_view_on]
-other = "在 GitHub 上查看"
+[layouts_docs_glossary_aka]
+other = "亦称作"
 
-[main_github_create_an_issue]
-other = "创建 GitHub issue"
+[layouts_docs_glossary_description]
+other = "此术语表旨在提供Kubernetes术语的完整、标准列表。其中包含特定于 Kubernetes 的技术术语以及能够构造有用的语境的一般性术语。"
```
Comment (xiangpengzhao) [inline i18n/zh.toml]: 第一个 Kubernetes 两侧加空格 :)

File: i18n/zh.toml [hunk 4]
```diff
@@ -63,56 +148,41 @@ other = "浏览往期的周报"
...
-[community_slack_name]
-other = "Slack"
-[community_stack_overflow_name]
-other = "Stack Overflow"
-[community_forum_name]
-other = "Forum"
-[community_events_calendar]
-other = "Events Calendar"
+[warning]
+other = "Warning:"
```
Comment (xiangpengzhao) [inline i18n/zh.toml]: 这个不是专有术语，是否也翻译一下？

File: i18n/zh.toml [hunk 5]
```diff
@@ -1,59 +1,144 @@
...
+other = """我们是 <a href="https://cncf.io/">CNCF</a> 毕业项目</p>"""
 
 [main_community_explore]
 other = "了解社区"
 
+[main_contribute]
+other = "贡献"
+
+[main_copyright_notice]
+other = """Linux 基金会&reg;。保留所有权利。Linux Foundation 已注册并使用商标。如需了解 Linux 基金会的商标列表，请访问<a href="https://www.linuxfoundation.org/trademark-usage" class="light-text">商标使用页面</a>"""
```
Comment (xiangpengzhao) [inline i18n/zh.toml]: 中间的 `Linux Foundation` 是否也把“基金会”翻译出来，保持统一？

**Comment by tengqm** [comment]
@xiangpengzhao All comments addressed.

**Comment by xiangpengzhao** [comment]
sorry for delay :)
/lgtm
/approve

**Pull request merged**